csrankings.csv

```diff
@@ -6358,8 +6361,7 @@ Girish Varma,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/girish.varma,Y
 Gisele L. Pappa,UFMG,http://homepages.dcc.ufmg.br/~glpappa,C_0ZLuYAAAAJ
 Gisele Lobo Pappa,UFMG,http://homepages.dcc.ufmg.br/~glpappa,C_0ZLuYAAAAJ
 Gita Alaghband,University of Colorado - Denver,http://cse.ucdenver.edu/~gita,NOSCHOLARPAGE
-Gita Reese Sukthankar,University of Central Florida,http://www.eecs.ucf.edu/~gitars,NOSCHOLARPAGE
-Gita Sukthankar,University of Central Florida,http://www.eecs.ucf.edu/~gitars,NOSCHOLARPAGE

```

Comment:
Needs to be present; same issue as above (alias). https://dblp.uni-trier.de/pers/s/Sukthankar:Gita_Reese.html